### PR TITLE
Cotr 53 implemet player camera movement

### PR DESCRIPTION
**Pull request by Elijuszek**

### CODE REVIEW

File: Scenes/Actors/Actor.cs [hunk 1]
```diff
@@ -8,9 +8,9 @@ public partial class Actor : CharacterBody2D
 {
     public Stats Stats { get; set; }
     public Team Team { get; set; }
```
Comment (Karolinskis) [inline Scenes/Actors/Actor.cs:10]: Add comments, describing what these variables do

File: Scenes/Actors/Player/Player.cs [hunk 2]
```diff
@@ -299,4 +298,8 @@ private void AttackTimerTimeout()
         //    Deliver();
         //}
     }
+    public void SetCameraTransform(NodePath cameraPath)
```
Comment (Karolinskis) [inline Scenes/Actors/Player/Player.cs:301]: Add summary

**Review by Karolinskis — CHANGES_REQUESTED** [review]

File: Scenes/Actors/Player/Player.cs (hunk 2 above)
Comment (Elijuszek) [reply Scenes/Actors/Player/Player.cs:305]: Done

**Review by Karolinskis — APPROVED** [review]
LGTM

**Pull request merged**